src/nginx/conf/nginx.conf

```diff
@@ -5,6 +5,7 @@ events {
 http {
     server {
         error_log logs/error.log debug;
+        underscores_in_headers on;
```

Comment:
Not here.  It should be here
https://github.com/cloudendpoints/endpoints-tools/blob/master/start_esp/nginx-auto.conf.template#L79

Prefer to add a flag to control it, and default to not supporting